### ISSUE TITLE
Load Layers - world and Aus layers in BCCVL

### ISSUE THREAD

**Issue by M-Nicholls** [AtlasOfLivingAustralia/layers-service#82]
From an email from BCCVL, they have a bunch of new climate layers.  Check if we don't have any of these and if they would be good to have in the ALA as well

•	Global datasets 
o	CRUCLIM (Past and Current: 5 year intervals from 1915 - 1995, 19 bioclimatic variables)
o	WorldClim (Current and Future: 1950-2000, 2050, 2070, 19 bioclimatic variables)
o	CLIMOND (Current and Future: 1975, 2030, 2050, 2080, 2090 & 2100, 35 bioclimatic variables)
•	Australia-wide datasets 
o	AustCLIM (Current and Future: 1976-2005, 10 year intervals 2015 - 2085, 19 bioclimatic variables)
•	Regional Australia datasets 
o	NARCLIM (Current and Future: 2000, 2030 & 2070, 25 bioclimatic variables. NSW, VIC and ACT)
o	TASCLIM (Past, Current and Future: 5 year intervals, 1975 - 2085, 19 bioclimatic variables, Tasmania)
o	AccuCLIM (Past and Current: 5 year intervals, 1965 - 2000, 7 bioclimatic variables. Aust Wet Tropics)

**Comment by ansell** [comment]
WorldClim-2.0 just released:

http://worldclim.org/version2

**Comment by Tasilee** [comment]
Yes, we need to REPLACE our current V1 versions (30sec - ~1km) with V2 when able.

We don't want to replicate BCCVL on climate scenarios but it would be good to have AustCLIM suite to enable simple (scatterplot) support.

**Comment by ansell** [comment]
The WorldClim-1.4 datasets are licensed under CC-BY-SA-4.0-International http://www.worldclim.org/current , where "current" still refers to 1.4 on the website at this point in time. 

However, that doesn't appear to be the case for WorldClim-2.0, which seems to be licensed as "You are not allowed to redistribute these data." http://worldclim.org/version2

The license situation will need more investigation before loading version 2

**Comment by Tasilee** [comment]
As discussed with Miles, we need to review and update the mess of licence info in the layers DB. Also - we don't redistribute the layers, only some values from them. Subtle, but maybe relevant.

**Comment by Tasilee** [comment]
Given layer rationalizations, suggest we download only

http://biogeo.ucdavis.edu/data/worldclim/v2.0/tif/base/wc2.0_30s_bio.zip
